### PR TITLE
doc: command hooks file management changes

### PR DESCRIPTION
**Pull request by lazpavel**
_Issue #, if available:_
CLI issue: https://github.com/aws-amplify/amplify-cli/issues/10253
CLI PR: https://github.com/aws-amplify/amplify-cli/pull/10667

_Description of changes:_
doc changes related to no longer pushing/pulling hook files to the s3 deployment bucket

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by offlineprogrammer — DISMISSED** [review]

**Review by josefaidt — APPROVED** [review]

File: src/pages/cli/project/command-hooks.mdx [hunk 1]
```diff
@@ -46,15 +46,15 @@ Minimum CLI version requirement met.
 ```
```
Comment (swaminator) [inline src/pages/cli/project/command-hooks.mdx:47]: Please note: Hooks need to be checked into source control in order to be made available across machines. `amplify pull` will **not** re-create the `hooks` folder automatically as these are considered changes that need to be committed to the repository.

File: src/pages/cli/project/command-hooks.mdx [hunk 2]
```diff
@@ -228,32 +228,51 @@ Dependency directories and files, such as `node_modules`, should be added to `ig
...
+  .then((event) => hookHandler(event.data, event.error))
+  .catch((err) => {
+    console.error(err);
+    process.exitCode = 1;
+  });
+```
+
 ## Using command hooks in Amplify's CI/CD pipeline
-Command hooks are executed when [CI/CD builds are triggered on Amplify Console](https://docs.aws.amazon.com/amplify/latest/userguide/multi-environments.html). To execute hook scripts in the Amplify Console, add the hook scripts to the `amplify/hooks` directory and run `amplify push` locally to upload the hooks.
+Command hooks are executed when [CI/CD builds are triggered on Amplify Console](https://docs.aws.amazon.com/amplify/latest/userguide/multi-environments.html). To execute hook scripts in the Amplify Console, add the hook scripts to the `amplify/hooks` directory and commit them to your source control.
```
Comment (swaminator) [inline src/pages/cli/project/command-hooks.mdx:262]: Replace Amplify Console with Amplify Hosting everywhere

**Review by swaminator — CHANGES_REQUESTED** [review]

File: src/pages/cli/project/command-hooks.mdx [hunk 3]
```diff
@@ -45,16 +45,18 @@ Minimum CLI version requirement met.
 ----- 🪝 pre-push execution end -----
 ```
 
+Please note: Hooks need to be checked into source control in order to be made available across machines. amplify pull will not re-create the hooks folder automatically as these are considered changes that need to be committed to the repository.
```
Comment (swaminator) [inline src/pages/cli/project/command-hooks.mdx:48]: Can we add this as an alert window under the title `Add command Hooks`

Comment (lazpavel) [reply src/pages/cli/project/command-hooks.mdx]: updated

**Review by jhockett — APPROVED** [review]

**Review by swaminator — APPROVED** [review]

**Pull request merged**